src/views.coffee

```diff
@@ -112,6 +112,17 @@ FormRenderer.Views.NonInputResponseField = FormRenderer.Views.ResponseField.exte
     @$el.html JST['partials/non_input_response_field'](@)
     @
 
+FormRenderer.Views.ResponseFieldPrice = FormRenderer.Views.ResponseField.extend
+  field_type: 'price'
+  events:
+    'blur [data-rv-input="model.value.cents"]': 'formatCents'
+
+  formatCents: (ev) ->
+    @$input = $(ev.target)
```

Comment:
1) the best-practice is to name the event parameter `e`, not `ev`.
2) you're setting `@$input` as an instance variable, when it should just be a local